mantaray/received.py

```diff
@@ -208,6 +209,9 @@ def _handle_nick(server_view: views.ServerView, old_nick: str, args: list[str])
             )
             if isinstance(view, views.ChannelView):
                 view.userlist.change_nick(old_nick, new_nick)
+
+        server_view.core.send(f"WHOIS {new_nick}")
```

Comment:
It isn't necessary to have this in `_handle_nick()`, because `get_nickmask()` uses `core.nick` which was updated above.